nifi/stackable/patches/1.21.0/002-NIFI-no-host-header-check-1.21.0.patch

```diff
@@ -0,0 +1,58 @@
+Subject: [PATCH] Allow bypassing check for host header.
```

Comment:
Attention: We don't build NiFi 1.21 from source, so this patch files have no effect (thx @maltesander for pointing this out).
Can you please place a README or similar in this folder instead that explains that you can not patch 1.21?

Reply:
Do we plan on building it from source at some point? Otherwise I'll just remove the patch instead, no need to confuse anybody if we never plan on supporting it.

Reply:
> Otherwise I'll just remove the patch instead

That was exactly my suggestion ;) With one addition of instead having a README stating this folder is not used